isubit/suitcase_interim
# INTERIM-103 Make nameplate logo alt text unique

To test
- Build either a new site locally or use a local copy of an existing Luggage site with suitcase_interim
- If using a copy of an existing site, first checkout this branch. Clear cache. Go to `admin/appearance/settings/suitcase_interim` and click the `Revert theme settings` button. View the home page and verify the nameplate logo is using the new text. If the site doesn't have this button, use `drush vset suitcase_interim_config_isu_nameplate_alt_text 'Iowa State University Logo'`
